.travis.yml

```diff
@@ -37,6 +37,11 @@ matrix:
   - os: linux
     compiler: gcc
     env: RUN_TESTS_ARGS="--cross" MESON_ARGS="--unity=on"
+  - os: linux
+    name: CheckUnusedReturnMissing
+    compiler: gcc
+    script: python run_project_tests.py --only cmake common fortran platform-linux
```

Comment:
This should probably be `python3`. The `python` executable points to Python 2.

Reply:
- @![pp_my_qrcode_1701038665878.jpg](https://github.com/mesonbuild/meson/assets/96654187/e2528bac-7d9e-4102-80de-698094407d9d)

Reply:
@  printf(\"Hello World\");